src/e2e-app/main-aot.ts

```diff
@@ -0,0 +1,4 @@
+import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
+import {E2eAppModuleNgFactory} from './e2e-app-module.ngfactory';
+
+platformBrowserDynamic().bootstrapModuleFactory(E2eAppModuleNgFactory);
```

Comment:
Yeah I think @josephperrott also had an issue about it but I didn't assign it to myself so I lost track of it. Changed it now.